docs/development/build-instructions-linux.md

```diff
@@ -73,7 +73,14 @@ Downloading certain files can take a long time. Notice that we are using
 
 ```sh
 $ cd electron
-$ ./script/bootstrap.py --verbose
+$ npm run bootstrap
+```
+
+You can also bootstrap for debugging:
+
+```sh
+$ cd electron
+$ npm run bootstrap-dev
```

Comment:
I'm a fan of the `bootstrap`, `bootstrap:debug`, `bootstrap:release` style of doing _sub_ npm scripts, thoughts?

Reply:
oh i like this too, i'll change it